### ISSUE TITLE
Build failed due to mismatch type error on android termux

### ISSUE THREAD

**Issue by hnmn** [FillZpp/sys-info-rs#64]
I was trying to build deno from source 
Using `cargo` but sys-info gives 
Following error

```rust
error[E0308]: mismatched types
   --> /data/data/com.termux/files/home/.cargo/registry/src/github.com-1ecc6299db9ec823/sys-info-0.7.0/lib.rs:618:22
    |
618 | pub fn boottime() -> Result<timeval, Error> {
    |        --------      ^^^^^^^^^^^^^^^^^^^^^^ expected enum `std::result::Result`, found `()`
    |        |
    |        implicitly returns `()` as its body has no tail or `return` expression
    |
    = note:   expected enum `std::result::Result<libc::timeval, Error>`
            found unit type `()`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
The following warnings were emitted during compilation:

warning: c/linux.c:96:11: warning: implicitly declaring library function 'index' with type 'char *(const char *, int)' [-Wimplicit-function-declaration]
warning:                 mount = index(procline, ' ');
warning:                         ^
warning: c/linux.c:96:11: note: include the header <strings.h> or explicitly provide a declaration for 'index'
warning: 1 warning generated.

error: could not compile `sys-info`.
```
I'm on `aarch64`

**Comment by tomap** [comment]
Hi, I just ran into an issue while trying to build deno from sources

error[E0308]: mismatched types
   --> /data/data/com.termux/files/home/.cargo/registry/src/github.com-1ecc6299db9ec823/sys-info-0.7.0/lib.rs:618:22
    |                                                                 618 | pub fn boottime() -> Result<timeval, Error> {                       |        --------      ^^^^^^^^^^^^^^^^^^^^^^ expected enum `std::result::Result`, found `()`                                               |        |                                                            |        implicitly returns `()` as its body has no tail or `return` expression                                                             |                                                                     = note:   expected enum `std::result::Result<libc::timeval, Error>`
            found unit type `()`                                                                                                            warning: unused doc comment                                             --> /data/data/com.termux/files/home/.cargo/registry/src/github.com-1ecc6299db9ec823/lexical-core-0.7.4/src/ftoa/ryu.rs:25:1                 |
25 | /// `d` must be non-special (NaN or infinite), non-negative,
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ rustdoc does not generate documentation for macro invocations                |
   = help: to document an item produced by a macro, the macro must produce the documentation as part of its expansion                                                                                             error: aborting due to previous error; 1

Not sure I pasted enough information. If you need more, please ask

**Comment by JOT85** [comment]
I believe this was fixed by #70?
If so, can this issue be closed?

**Comment by tomap** [comment]
Maybe publish a new version?
